src/xmlutil.rs

```diff
@@ -250,4 +282,23 @@ mod tests {
         }
     }
 
+    #[test]
+    fn skip_tree_with_subtree() {
+        let file = File::open("tests/sample-data/skip_tree.xml").unwrap();
+        let file = BufReader::new(file);
+        let mut my_parser  = EventReader::new(file);
+        let my_stack = my_parser.events().peekable();
+        let mut reader = XmlResponseFromFile::new(my_stack);
+
+        // skip two leading fields since we ignore them (xml declaration, return type declaration)
+        reader.next();
+        reader.next();
```

Comment:
I think this second `.next()` call is skipping the `<a>` tag.  This test may need to skip only one section, the xml declaration.